RELEASE-NOTES.txt

```diff
@@ -4,6 +4,16 @@
 * [*] [internal] Refactor sending the API requests for searching posts and pages. [#21976]
 * [*] Fix an issue in Menu screen where it fails to create default menu items. [#21949]
 * [*] [internal] Refactor how site's pages are loaded in Site Settings -> Homepage Settings. [#21974]
+* [***] Update the design of the Site Media screen with an improved selection mode, updated context menus, a way to share more than one item at a time, better support for animated GIFs, fix a couple of visual issues with state views and search, and more [#21457]
```

Comment:
Might be nice to prefix with "Site Media:" for the copy editors

Reply:
Good point. I rebased and saw the Posts & pages notes and updated it to follow suit.